Source/Core/Core/Movie.cpp

```diff
@@ -202,6 +202,12 @@ std::string GetRTCDisplay()
   return format_time.str();
 }
 
+// NOTE: GPU Thread
+std::string GetRerecords()
+{
+  return std::format("Rerecords: {}", s_rerecords);
```

Comment:
Dolphin targets C++17 currently, where `std::format` doesn't exist; however, `std::format` is just a standardization of the fmt library, which is used by dolphin.  So `return fmt::format("Rerecords: {}", s_rerecords);` should work fine.

The visual studio solution doesn't restrict it to C++17, which is probably why you didn't run into this in development.

Reply:
Ah thank you, I didn't realize this. I will keep this in mind for the future.